home_page.php

```diff
@@ -59,8 +59,8 @@
 			</header>
 			
 			<!-- Section 1: Background image display -->
-			<div class="section1"style = "background-color:gray;">
-				<img src="">
```

Comment:
This was a placeholder for the background image, which was to be populated later